bug-pattern/src/main/java/jp/skypencil/findbugs/slf4j/IllegalPassedClassDetector.java

```diff
@@ -71,7 +85,8 @@ private void memorizeResultOfGetClassMethod(int code) {
     returnedClass.setUserValue(classOfCaller);
   }
 
-  private JavaType findClass(String storedClassName) {
+  @VisibleForTesting
+  JavaType findClass(@SlashedClassName String storedClassName) throws ClassFormatException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'org.apache.bcel.classfile.ClassFormatException' which is a runtime exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)